models/articles.model.js

```diff
@@ -6,9 +6,17 @@ const articles = require("../db/data/test-data/articles");
 module.exports.fetchArticleById = (id) => {
   return db
     .query(
-      `SELECT * FROM articles 
-        WHERE article_id = $1`,
+      `SELECT articles.*,
+    COUNT(comments.comment_id) AS comment_count 
+    FROM articles
+    LEFT JOIN comments on articles.article_id = comments.article_id
+    WHERE articles.article_id = $1 
+    GROUP BY articles.article_id
+    `,
```

Comment:
Nice! For extra brownie points when you use COUNT it does add the number as a string data type- there is a way in the query to cast the value as a number. 

You don't have to do it, but it would save you having to manipulate it in the front end project, worth a look at some point!